library/graph_data_lib.py

```diff
@@ -121,28 +118,13 @@ def get_graph_data(client, category, product_name):
 
 if __name__ == "__main__":
 
-  client = "mongodb://root:1000Sunny@externaltaotarodb1.mongodb.rds.aliyuncs.com:3717,externaltaotarodb2.mongodb.rds.aliyuncs.com:3717/admin?authSource=admin&replicaSet=mgset-28314303&readPreference=primary&ssl=false"
-  test_product = 'Athena Crop Top'
-  test_scatter, test_bar, test_swarm, test_revenue, test_pie=get_graph_data(client, 'Tops', test_product)
-  print(test_scatter)
 
+  # client = "mongodb://root:1000Sunny@externaltaotarodb1.mongodb.rds.aliyuncs.com:3717,externaltaotarodb2.mongodb.rds.aliyuncs.com:3717/admin?authSource=admin&replicaSet=mgset-28314303&readPreference=primary&ssl=false"
```

Comment:
remove it completely please, not just comment out. it's not just a matter of using it, but also a security issue because now anyone can see it online once you commit to git history. please remove the username and password from all the git history. you can look this up on google on how to do so

Reply:
i would be ok/lenient with it if its not our root user. if its our test account, that is fine because the test account has restricted permissions and can't touch anything else. but root can do and view everything